launch/openHAB2.setup

```diff
@@ -2295,8 +2295,8 @@
   </setupTask>
   <setupTask
       xsi:type="jdt:JRETask"
-      version="JavaSE-1.8"
-      location="${jre.location-1.8}"/>
+      version="JavaSE-11"
```

Comment:
This means you can't setup the eclipse IDE for openHAB 2.5.x development anymore... Or is it possible to select the openHAB2 setup for the 2.5.x branch from the eclipse installer???

Reply:
Perhaps @kaikreuzer knows if it is possible to configure the installer to use the `openHAB2.setup` of the 2.5.x branch in some way?

Perhaps it can be configure on a website or if we change the default branch in openhab-distro to 2.5.x?

If that's not possible we should change this version back to prevent issues. The filename itself can also use an update since there's still a 2 in it.

Reply:
No, the installer uses exactly this file from master only.
What might work is to move the jdk setupTask into a project and then have two different "openHAB Development" projects for 2.5 and 3.0.

Reply:
I see it should work again now https://github.com/openhab/openhab-distro/pull/1063 is merged and I found the openHAB2.setup URL is part of the [com.github.projects.setup](https://git.eclipse.org/c/oomph/org.eclipse.oomph.git/tree/setups/com.github.projects.setup#n260) file in the Oomph installer. :male_detective:

See also [Contributing to a Project Catalog](https://wiki.eclipse.org/Eclipse_Oomph_Authoring#Contributing_to_a_Project_Catalog).